contracts/BodhiToken.sol

```diff
@@ -102,12 +102,22 @@ contract BodhiToken is StandardToken, Ownable {
 
   /// @notice Shows the amount of BOT the user will receive for amount of exchanged wei
   /// @param _weiAmount Exchanged wei amount to convert
+  /// @param _nativeDecimals Number of decimals of the token being exchange for BOT
+  /// @param _decimals Number of decimals of BOT token
+  /// @param _exchangeRate Number of BOT per exchange token
```

Comment:
keep order the same with definition below

Reply:
fixed. merging after tests pass.